### ISSUE TITLE
Fine-Tuning on a Small Dataset

### ISSUE THREAD

**Issue by said702** [dailenson/One-DM#35]
First of all, thank you for your excellent work and for providing this repository. It is a great resource!

I am currently trying to fine-tune the English model using a small dataset with 200 samples to improve its performance on specific words that are not generated correctly. However, after fine-tuning the model for a few epochs, the generated images are almost completely black and unrecognizable.

Is this issue caused by using too few samples?

Thank you in advance for your guidance!

**Comment by dailenson** [comment]
Hi, Thanks for your attention! Before fine-tuning, you could try to turn down the learning rate by 10 times.

**Comment by said702** [comment]
Thank you very much for the quick response! One more question, please: The English one-dm model that you kindly released was trained on both the CVL and IAM datasets, correct?

**Comment by nguyenthuy1681999** [comment]
@dailenson Can your model generate handwriting in lines? If so what preparations are needed? Thank you
